Memory/Hooks.cpp

```diff
@@ -270,7 +269,7 @@ void* Hooks::Player_tickWorld(C_Player* _this, __int64 unk) {
 
 	if (_this == g_Data.getLocalPlayer()) {
 		// TODO: refactor all modules to not use GameMode
-		C_GameMode* gm = *reinterpret_cast<C_GameMode**>(reinterpret_cast<__int64>(_this) + 4840);
+		C_GameMode* gm = *reinterpret_cast<C_GameMode**>(reinterpret_cast<__int64>(_this) + 0x1248);
```

Comment:
Change the 0x1248 to 1248 it looks cleaner

Reply:
Dont think that would work, changing it to 4680 might

Reply:
Could someone confirm this ^^